src/executor/ObjectDetector.py

```diff
@@ -0,0 +1,13 @@
+from executor.iExecutor import iExecutor
+from service import ModelManager
+
+class ObjectDetector(iExecutor):
+
+    def __init__(self, *kargs, **kwargs):
+        super().__init__(*kargs, **kwargs)
+
+    @classmethod
+    def run(self, item):
+        network = ModelManager.get_model("faster_rcnn_resnet101_kitti_2018_01_28")
+        ids, scores, bboxes, masks = [X[0].asnumpy() for X in network(item.npy)]
+        breakpoint()
```

Comment:
Is this function supposed to return something? How do we get the data in ids, scores, bboxes, and masks?

Also, there's a breakpoint (up to you if you want to leave it in)

Reply:
This is an executor. Executors don't return anything. They just modify the item dictionary

Reply:
I haven't been able to test this because I can't run the pipeline still, which is why there is currently a breakpoint